rubocop.yml

```diff
@@ -29,15 +32,14 @@ Metrics/CyclomaticComplexity:
 Metrics/LineLength:
   Max: 120
 Metrics/MethodLength:
-  Enabled: false
+  Enabled: true
+  Max: 20 # double the default value
```

Comment:
too intrusive in my opinion and not doable in some projects. it's also debateable if 3 x 20 lines methods are better than 1 x 60 lines method *if the split is not done properly*. and Rubocop doesn't tell you how to split it, it only tells you that *you have to split it*.